### DISCUSSION TITLE
Utilização de parte do saldo positivo de eventos em edições futuras.

### DISCUSSION THREAD

**Discussion by meu-bem** [apyb/comunidade#115] [Geral]
Atualmente, o [guia de como utilizar a infra da APyB](https://apyb.python.org.br/guias/como-organizar-eventos/) dita:
> [...] qualquer saldo positivo que restar do evento ficará como doação para a APyB custear sua operação e servir como fundo de reserva para a organização da Python Brasil. Portanto nenhum saldo positivo poderá ser repassado ou transferido para outros eventos.
No entanto, seria de grande ajuda para iniciar os eventos se fosse possível contar com parte do caixa de edições anteriores.

Semelhante ao referido na [issue](https://github.com/apyb/comunidade/discussions/29) da @biancarosa.

## Proposta:
* Ao término do evento 50% do saldo positivo já seria direcionado em forma de doação para a APyB, como acontece atualmente.
* Os outros 50% poderiam ser utilizados pelo evento em edições seguintes, já que muitas vezes auxílios como os da PSF podem levar tempo para serem recebidos, o que pode atrasar algumas das produções do evento.

**Comment by rougeth** [comment]
Já temos algo assim implementado. 50% desse valor vai para o fundo de inclusão: https://python.org.br/fundo-para-inclusão/

Caso o evento precise adiantar algum valor que irá receber no futuro, a organização pode entrar em contado com a diretoria da APyB. Dependendo da situação, pode ser que a APyB consiga ajudar.

**Reply by meu-bem** [reply]
@rougeth, que massa saber disso!
Com quem a gente pode falar pra conseguir adiantar esse recurso?

**Reply by rougeth** [reply]
@apyb/diretoria

**Reply by tiidadavena** [reply]
Obrigade por nos marcar, Rougeth!

**Comment by tiidadavena** [comment]
> Atualmente, o [guia de como utilizar a infra da APyB](https://apyb.python.org.br/guias/como-organizar-eventos/) dita:
> 
> > [...] qualquer saldo positivo que restar do evento ficará como doação para a APyB custear sua operação e servir como fundo de reserva para a organização da Python Brasil. Portanto nenhum saldo positivo poderá ser repassado ou transferido para outros eventos.
> > No entanto, seria de grande ajuda para iniciar os eventos se fosse possível contar com parte do caixa de edições anteriores.
> 
> Semelhante ao referido na [issue](https://github.com/apyb/comunidade/discussions/29) da @biancarosa.
> 
> ## Proposta:
> * Ao término do evento 50% do saldo positivo já seria direcionado em forma de doação para a APyB, como acontece atualmente.
> * Os outros 50% poderiam ser utilizados pelo evento em edições seguintes, já que muitas vezes auxílios como os da PSF podem levar tempo para serem recebidos, o que pode atrasar algumas das produções do evento.

Boa tarde, @MateusSantosMeuBem ! Você está organizando a Python Brasil 2024 ou outro evento da comunidade Python do Brasil?

**Reply by meu-bem** [reply]
Oi, @tiidadavena. Tudo bem? Estou sim, é a Python Norte.
